pkg/apis/apps/types.go

```diff
@@ -532,7 +532,7 @@ type RollingUpdateDaemonSet struct {
 	// The maximum number of DaemonSet pods that can be unavailable during the
 	// update. Value can be an absolute number (ex: 5) or a percentage of total
 	// number of DaemonSet pods at the start of the update (ex: 10%). Absolute
-	// number is calculated from percentage by rounding down to a minimum of one.
+	// number is calculated from percentage by rounding up with a minimum of one.
```

Comment:
I think "rounded up." is sufficient.  The "minimum of one" is part of rounding up for every case except `maxUnavailable: 0`, and the zero case is discussed explicitly in the line below this.

Reply:
Ironically this line is in here because someone asked.  I would leave it for now, since I'm probably going to go start the process to change the behavior.